components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/listener/SCIMUserOperationListener.java

```diff
@@ -67,6 +69,7 @@
 public class SCIMUserOperationListener extends AbstractIdentityUserOperationEventListener {
 
     private static final Log log = LogFactory.getLog(SCIMUserOperationListener.class);
+    private static final String VALUE_SEPARATOR = ",";
```

Comment:
shall we use the MultiAttributeSeperator constant in kernel?

Reply:
Fixed with 06768cde82fd685e208bdf2a3fdbbaaa5649ec71